CRITICAL - 2021-07-06 00:12:14 --> Class name must be a valid object or a string
#0 C:\xampp\htdocs\project-dashboard\vendor\codeigniter4\framework\system\Config\BaseService.php(263): Myth\Auth\Config\Services::activator(Object(Myth\Auth\Config\Auth), false)
#1 C:\xampp\htdocs\project-dashboard\vendor\codeigniter4\framework\system\Config\BaseService.php(192): CodeIgniter\Config\BaseService::__callStatic('activator', Array)
#2 C:\xampp\htdocs\project-dashboard\vendor\myth\auth\src\Config\Services.php(86): CodeIgniter\Config\BaseService::getSharedInstance('activator', NULL)
#3 C:\xampp\htdocs\project-dashboard\vendor\codeigniter4\framework\system\Config\BaseService.php(263): Myth\Auth\Config\Services::activator()
#4 C:\xampp\htdocs\project-dashboard\vendor\codeigniter4\framework\system\Common.php(1084): CodeIgniter\Config\BaseService::__callStatic('activator', Array)
#5 C:\xampp\htdocs\project-dashboard\vendor\myth\auth\src\Controllers\AuthController.php(195): service('activator')
#6 C:\xampp\htdocs\project-dashboard\vendor\codeigniter4\framework\system\CodeIgniter.php(928): Myth\Auth\Controllers\AuthController->attemptRegister()
#7 C:\xampp\htdocs\project-dashboard\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
#8 C:\xampp\htdocs\project-dashboard\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\project-dashboard\public\index.php(37): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\project-dashboard\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2021-07-06 01:35:20 --> Invalid file: Views\auth\login.php
#0 C:\xampp\htdocs\project-dashboard\vendor\codeigniter4\framework\system\View\View.php(213): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('Views\\auth\\logi...')
#1 C:\xampp\htdocs\project-dashboard\vendor\codeigniter4\framework\system\Common.php(1271): CodeIgniter\View\View->render('Views\\auth\\logi...', Array, true)
#2 C:\xampp\htdocs\project-dashboard\vendor\myth\auth\src\Controllers\AuthController.php(429): view('Views\\auth\\logi...', Array)
#3 C:\xampp\htdocs\project-dashboard\vendor\myth\auth\src\Controllers\AuthController.php(57): Myth\Auth\Controllers\AuthController->_render('Views\\auth\\logi...', Array)
#4 C:\xampp\htdocs\project-dashboard\vendor\codeigniter4\framework\system\CodeIgniter.php(928): Myth\Auth\Controllers\AuthController->login()
#5 C:\xampp\htdocs\project-dashboard\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
#6 C:\xampp\htdocs\project-dashboard\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\project-dashboard\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\project-dashboard\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#9 {main}
CRITICAL - 2021-07-06 01:36:01 --> Invalid file: \Views\auth\login.php
#0 C:\xampp\htdocs\project-dashboard\vendor\codeigniter4\framework\system\View\View.php(213): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('\\Views\\auth\\log...')
#1 C:\xampp\htdocs\project-dashboard\vendor\codeigniter4\framework\system\Common.php(1271): CodeIgniter\View\View->render('\\Views\\auth\\log...', Array, true)
#2 C:\xampp\htdocs\project-dashboard\vendor\myth\auth\src\Controllers\AuthController.php(429): view('\\Views\\auth\\log...', Array)
#3 C:\xampp\htdocs\project-dashboard\vendor\myth\auth\src\Controllers\AuthController.php(57): Myth\Auth\Controllers\AuthController->_render('\\Views\\auth\\log...', Array)
#4 C:\xampp\htdocs\project-dashboard\vendor\codeigniter4\framework\system\CodeIgniter.php(928): Myth\Auth\Controllers\AuthController->login()
#5 C:\xampp\htdocs\project-dashboard\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
#6 C:\xampp\htdocs\project-dashboard\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\project-dashboard\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\project-dashboard\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#9 {main}
CRITICAL - 2021-07-06 01:39:59 --> Invalid file: ..\app\Views\auth\login.php
#0 C:\xampp\htdocs\project-dashboard\vendor\codeigniter4\framework\system\View\View.php(213): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('..\\app\\Views\\au...')
#1 C:\xampp\htdocs\project-dashboard\vendor\codeigniter4\framework\system\Common.php(1271): CodeIgniter\View\View->render('..\\app\\Views\\au...', Array, true)
#2 C:\xampp\htdocs\project-dashboard\vendor\myth\auth\src\Controllers\AuthController.php(429): view('..\\app\\Views\\au...', Array)
#3 C:\xampp\htdocs\project-dashboard\vendor\myth\auth\src\Controllers\AuthController.php(57): Myth\Auth\Controllers\AuthController->_render('..\\app\\Views\\au...', Array)
#4 C:\xampp\htdocs\project-dashboard\vendor\codeigniter4\framework\system\CodeIgniter.php(928): Myth\Auth\Controllers\AuthController->login()
#5 C:\xampp\htdocs\project-dashboard\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
#6 C:\xampp\htdocs\project-dashboard\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\project-dashboard\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\project-dashboard\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#9 {main}
CRITICAL - 2021-07-06 02:51:15 --> Class name must be a valid object or a string
#0 C:\xampp\htdocs\project-dashboard\vendor\codeigniter4\framework\system\Config\BaseService.php(263): Myth\Auth\Config\Services::activator(Object(Myth\Auth\Config\Auth), false)
#1 C:\xampp\htdocs\project-dashboard\vendor\codeigniter4\framework\system\Config\BaseService.php(192): CodeIgniter\Config\BaseService::__callStatic('activator', Array)
#2 C:\xampp\htdocs\project-dashboard\vendor\myth\auth\src\Config\Services.php(86): CodeIgniter\Config\BaseService::getSharedInstance('activator', NULL)
#3 C:\xampp\htdocs\project-dashboard\vendor\codeigniter4\framework\system\Config\BaseService.php(263): Myth\Auth\Config\Services::activator()
#4 C:\xampp\htdocs\project-dashboard\vendor\codeigniter4\framework\system\Common.php(1084): CodeIgniter\Config\BaseService::__callStatic('activator', Array)
#5 C:\xampp\htdocs\project-dashboard\vendor\myth\auth\src\Controllers\AuthController.php(195): service('activator')
#6 C:\xampp\htdocs\project-dashboard\vendor\codeigniter4\framework\system\CodeIgniter.php(928): Myth\Auth\Controllers\AuthController->attemptRegister()
#7 C:\xampp\htdocs\project-dashboard\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
#8 C:\xampp\htdocs\project-dashboard\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\project-dashboard\public\index.php(37): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\project-dashboard\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2021-07-06 02:58:37 --> Class name must be a valid object or a string
#0 C:\xampp\htdocs\project-dashboard\vendor\codeigniter4\framework\system\Config\BaseService.php(263): Myth\Auth\Config\Services::activator(Object(Myth\Auth\Config\Auth), false)
#1 C:\xampp\htdocs\project-dashboard\vendor\codeigniter4\framework\system\Config\BaseService.php(192): CodeIgniter\Config\BaseService::__callStatic('activator', Array)
#2 C:\xampp\htdocs\project-dashboard\vendor\myth\auth\src\Config\Services.php(86): CodeIgniter\Config\BaseService::getSharedInstance('activator', NULL)
#3 C:\xampp\htdocs\project-dashboard\vendor\codeigniter4\framework\system\Config\BaseService.php(263): Myth\Auth\Config\Services::activator()
#4 C:\xampp\htdocs\project-dashboard\vendor\codeigniter4\framework\system\Common.php(1084): CodeIgniter\Config\BaseService::__callStatic('activator', Array)
#5 C:\xampp\htdocs\project-dashboard\vendor\myth\auth\src\Controllers\AuthController.php(195): service('activator')
#6 C:\xampp\htdocs\project-dashboard\vendor\codeigniter4\framework\system\CodeIgniter.php(928): Myth\Auth\Controllers\AuthController->attemptRegister()
#7 C:\xampp\htdocs\project-dashboard\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
#8 C:\xampp\htdocs\project-dashboard\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\project-dashboard\public\index.php(37): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\project-dashboard\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#11 {main}
